workshops/templates/base.html

```diff
@@ -64,17 +68,29 @@
           </form>
 
           <ul class="nav navbar-nav navbar-right">
+            {% if perms.workshops.add_event or perms.workshops.add_host or perms.workshops.add_person or perms.workshops.add_airport %}
```

Comment:
Huh - post a question to Stack Overflow?  Seems like there *ought* to be 
a point where you could inject something that would add your own Boolean 
to perms.workshops before the template is rendered. (But it's a very low 
priority...)